### DISCUSSION TITLE
bug with holograms

### DISCUSSION THREAD

**Discussion by BaMJ9** [sgkoishi/yaaiomni#3] [General]
Does anyone have an idea on how to fix the problem with holograms remaining when a player disconnects? I've been having this problem for months and it causes a lot of lag

It is as if the player character stays there but is not really connected when relogging these holograms disappear

**Comment by sgkoishi** [comment]
I've received some reports related to the problem (after using this plugin), but probably it also happens without the plugin: Pryaxis/TShock#2948 and Pryaxis/TShock#2871. I can't reproduce it and not quite sure what is causing it 🤔 
For _some_ reason the player leave event didn't get triggered and thus the server didn't broadcast the leave to other players - probably won't cause any lag, but only prevent the same player from joining.

**Reply by sgkoishi** [reply]
Probably fixed 1dd1393, can you confirm? @BaMJ9
